packages/@sanity/imagetool/src/ImageTool.js

```diff
@@ -15,6 +16,41 @@ const MARGIN_PX = 8
 const CROP_HANDLE_SIZE = 12
 const HOTSPOT_HANDLE_SIZE = 10
 
+function checkCropBounderies(value, delta) {
```

Comment:
Typo bounderies should be boundaries (or just bounds for short)

Reply:
thx

Reply:
Typo still there?